default.conf

```diff
@@ -34,12 +45,30 @@ server {
     # /home
     # redirect to /app/assets/index.html
     location = /home {
-        add_header Cache-Control "no-cache";
+        if ($is_mobile = "1") {
+            return 301 $scheme://$http_host/mobilehome;
```

Comment:
I'm not sure if we want this to redirect to the pwa?

Reply:
to be honest, i'm not sure what `/mobilehome` is.  Is this a new route?  

does it lead to a Mobile friendly version of our Web app?  Or is it supposed to link to a specific PWA?  If so, how does it know which one?

Reply:
My mistake was thinking that PWA = mobile. This should actually link to the PWA. Can you suggest what this should be?

Reply:
https://github.com/digi-serve/ab_service_web/pull/11/commits/079d1fd3f62b4c04a87fd61c7b33c6fc9e4c668b